content/en/guides/diversity-inclusion.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Building an Inclusive Open Source Community
+---
+
+Diversity and inclusion are vital to the future of open source communities . Projects grow and thrive -- and perform better technically and economically – when contributors with diverse perspectives, experiences, and skill sets participate. Open source communities  seek to be open; this means they want everyone in the community who wants to contribute to feel welcome to do so regardless of gender, gender identity, sexual orientation, disability, race, ethnicity, age, religion, class origin, regional origin, educational background, political affiliation, or economic status. Open source communities can apply best practices to their communities that promote inclusiveness and diversity. To address these critical issues takes the strength of the collective community. Together, we can increase awareness, and subsequently address the representation gap—to benefit us all.
```

Comment:
nit: extra space before the period in the first sentence. Might be good to clean up if you need to do an update.